apps/gaia/src/pages/support.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from "react"
+import Head from "next/head"
+import Img from "react-optimized-image"
+import { useQueryString } from "../hooks/useQueryString"
+import LogoutIcon from "../icons/log-out.svg"
+import loadCanny from "../utils/canny"
+import useGetUser from "../hooks/api/useGetUser"
+import useGetChild from "../hooks/api/useGetChild"
+
+const SupportPage = () => {
```

Comment:
wtf why function can only have 25 lines of code

Reply:
yeah, functions were supposed to do only 1 thing, usually, if it has more than 25 lines, it is doing too many things at once. Doesn't apply to react I suppose, since components are long...

I haven't try to adapt codeclimate's settings to react properly yet wkwwk